pkg/bundle/sync.go

```diff
@@ -265,7 +264,7 @@ func (e pkcs12Encoder) encode(trustBundle string) ([]byte, error) {
 		})
 	}
 
-	return pkcs12.EncodeTrustStoreEntries(rand.Reader, entries, e.password)
+	return pkcs12.LegacyDES.EncodeTrustStoreEntries(entries, e.password)
```

Comment:
```suggestion
	return pkcs12.LegacyRC2.EncodeTrustStoreEntries(entries, e.password)
```

I think we should be conservative about changing the crypto in this PR. Ref. https://github.com/cert-manager/cert-manager/pull/6517